app/common/src/cljc/teet/authorization/authorization_check.cljc

```diff
@@ -14,6 +15,12 @@
                                  slurp
                                  read-string))))
 
+(defonce roles
```

Comment:
Maybe rename `all-roles`, doc string "A delay of a set of all role keywords used in the application." or some such.